src/services/conversationsService.js

```diff
@@ -236,11 +238,11 @@ class ConversationsService {
     store.dispatch(
       upsertChat({
         _id: selectedConversationId,
-        image_url: URL.createObjectURL(file),
+        image_url: isHeic(file.name) ? null : URL.createObjectURL(file),
       })
     );
 
-    const imageFile = await processFile(file);
+    const imageFile = await processFile(file, 0.2, 300);
```

Comment:
I see 1024 size when create group chat, and 300 when update. What's the diff?

Reply:
It's just a misprint, corrected to 300. This is the number of the maximum possible output file size